packages/core/src/scuttle.d.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Options for scuttling global properties
+ */
+export interface ScuttleOpts {
+  enabled: boolean;
+  exceptions?: Array<string | RegExp>;
+  scuttlerName?: string;
+}
+
+/**
+ * Reference to the global object
+ */
+export interface GlobalRef {
+  globalThis?: Record<string, unknown>;
```

Comment:
This is probably fine for our purposes (because it should a) allow us to remove properties and b) discourage us from modifying properties otherwise due to type `unknown`), but it begs the question: what happens in scuttling if we have a global symbol? [graceful-fs](https://npm.im/graceful-fs) does this.  or a numeric key, for that matter?